components/playlist/browser/media_detector_component_manager.cc

```diff
@@ -29,11 +60,16 @@ std::string ReadScript(const base::FilePath& path) {
   return contents;
 }
 
-const std::string& GetLocalScript() {
-  static const std::string kScript =
-      ui::ResourceBundle::GetSharedInstance().LoadDataResourceString(
-          IDR_PLAYLIST_MEDIA_DETECTOR_JS);
-  return kScript;
+base::flat_map<ScriptName, std::string> ReadScriptsFromComponent(
+    base::flat_set<base::FilePath> files) {
+  base::flat_map<ScriptName, std::string> script_map;
+  for (const auto& path : files) {
```

Comment:
@sangwoo108 
What it the typical number/size of these files?

Reply:
For now we have 2 scripts, and they are 5kb and 2kb respectively.

Reply:
@atuchin-m said that if a file becomes bigger than 1mb, then we should revisit this.

Reply:
It's okay for now. If the total size becomes more that 1MB, please ping me.